hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieBootstrapRelation.scala

```diff
@@ -188,11 +188,23 @@ case class HoodieBootstrapRelation(override val sqlContext: SQLContext,
 
   override def updatePrunedDataSchema(prunedSchema: StructType): HoodieBootstrapRelation =
     this.copy(prunedDataSchema = Some(prunedSchema))
+
+  def toHadoopFsRelation: HadoopFsRelation = {
+      HadoopFsRelation(
+        location = fileIndex,
+        partitionSchema = fileIndex.partitionSchema,
+        dataSchema = fileIndex.dataSchema,
+        bucketSpec = None,
+        fileFormat = fileFormat,
+        optParams)(sparkSession)
+  }
 }
 
 
 object HoodieBootstrapRelation {
 
+  val USE_FAST_BOOTSTRAP_READ = "hoodie.bootstrap.relation.use.fast.bootstrap.read"
```

Comment:
@jonvex : Can we just use one config hoodie.bootstrap.data.queries.only and get away with hoodie.bootstrap.relation.use.fast.bootstrap.read ?

Reply:
Ok. I was able to reuse the config. The reason I introduced the config was because I was worried about the case where the config is enabled but we decide to use the regular bootstrap read. I tested and made sure the config was correct in all places where it is used.